src/modules/checkBox.js

```diff
@@ -0,0 +1,11 @@
+window.handleCheckbox = (id) => {
+  const tasks = JSON.parse(localStorage.getItem('TasksInfo')) || [];
+  for (let i = 0; i < tasks.length; i += 1) {
+    if (id === tasks[i].id) {
+      tasks[i].completed = true;
+    }
+  }
+  localStorage.setItem('TasksInfo', JSON.stringify(tasks));
+};
+
+export default window.handleCheckbox;
```

Comment:
- [ ] Your checkbox toggle only works once when I toggle to `true`. The task does not toggle back to `false` when I uncheck the box. Kindly fix that.

- [ ] **[OPTIONAL]** Upon reloading the page, I think it would be a good idea to check the boxes of the tasks whose `completed` values are `true`